src/main/java/com/SAS/Controllers/systemController/SystemController.java

```diff
@@ -79,10 +79,12 @@ public String openAssociationSystem() {
      */
     public boolean addExternalSystem(String name) {
         if (name == null) {
+            logger.logEvent("Fault: a null external system tried to be added");
```

Comment:
You used logEvent function for the errors.
you need to used logError instead so it will be written to different files.
need to be changed in all places to logError

Reply:
change it at the latest commit